src/component/filter/filter.jsx

```diff
@@ -0,0 +1,108 @@
+import { useCart } from "../../context/cardContext"
```

Comment:
Not mandatory but good to know: You can use path alias to avoid moving back in folders using ../../
refer For Create React App without using Craco section in this https://dev.to/theodorusclarence/clean-up-your-imports-using-absolute-imports-alias-on-react-apps-next-js-cra-and-react-vite-2160

Reply:
_Thank You so much for this. Surely look into this_